codenvy-api-project/src/main/java/com/codenvy/api/project/server/ProjectService.java

```diff
@@ -258,10 +258,17 @@ public ProjectDescriptor createProject(@ApiParam(value = "ID of workspace to cre
             throws ConflictException, ForbiddenException, ServerException {
 
         final GeneratorDescription generatorDescription = newProject.getGeneratorDescription();
+        Map<String, String> options;
+        if (generatorDescription == null) {
+            options = Collections.emptyMap();
+        } else {
+            options = generatorDescription.getOptions();
+        }
+
```

Comment:
I think it's because developer are too dumb ;-) as we may end in complexe code (well in that case it's not too difficult but rules are for all the cases)

http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/coding/AvoidInlineConditionalsCheck.html